lib/asciidoctor-pdf/converter.rb

```diff
@@ -336,8 +336,11 @@ def convert_abstract node
     add_dest_for_block node if node.id
     pad_box @theme.abstract_padding do
       theme_font :abstract do
-        # FIXME control first_line_options using theme
-        prose_opts = { line_height: @theme.abstract_line_height, first_line_options: { styles: [font_style, :bold] } }
+        prose_opts = { line_height: @theme.abstract_line_height }
+        # FIXME control more first_line_options using theme
+        if (first_line_font_style = @theme.abstract_first_line_font_style.to_sym) && first_line_font_style != font_style
```

Comment:
Thanks!

The general rule of thumb with Ruby is to avoid method calls if there's another way to do it without one. So the nested if is the right choice in this case (or just call `.to_sym` both times)!